runner/src/metrics.ts

```diff
@@ -13,9 +13,23 @@ const EXECUTION_DURATION = new promClient.Gauge({
   labelNames: ['indexer', 'type'],
 });
 
+const CACHE_HIT_STREAMER_MESSAGE = new promClient.Counter({
+  name: 'redis_cache_hit_for_streamer_message',
+  help: 'The number of times the streamer message cache was hit',
+  labelNames: ['type']
+});
+
+const CACHE_MISS_STREAMER_MESSAGE = new promClient.Counter({
+  name: 'redis_cache_miss_for_streamer_message',
+  help: 'The number of times the streamer message cache was missed',
+  labelNames: ['type']
+});
```

Comment:
```suggestion
const CACHE_HIT = new promClient.Counter({
  name: 'queryapi_runner_cache_hit',
  help: 'Number of times cache hit was successful',
  labelNames: ['type', 'key']
});
```
It might make sense to make this more generic, and add a new dimension/label for the key which was hit/missed. That way we use this more globally.

Same for above

Reply:
That's a good idea. I'll implement it.